gapic/templates/%namespace/%name_%version/%sub/services/%service/client.py.j2

```diff
@@ -427,6 +429,26 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
             {{ service.client_name }}._compare_universes(self.universe_domain, self.transport._credentials))
         return self._is_universe_domain_valid
 
+    def _add_cred_info_for_auth_errors(
+        self,
+        error: core_exceptions.GoogleAPICallError
+    ) -> None:
+        """Adds credential info string to error details for 401/403/404 errors.
```

Comment:
Is this part of a client library AIP? https://google.aip.dev/client-libraries. If not, should we consider creating one? The AIPs are meant to help define requirements for client libraries which will also be used when we create client libraries for a new language like rust.